Cargo.toml

```diff
@@ -73,7 +67,7 @@ opt-level = 2
 inherits = "dev"
 opt-level = 1
 lto = false
-debug = "line-tables-only"
+# debug = "line-tables-only"
```

Comment:
This is safe to remove; it worked around a problem only applicable to very large Rust WASM artifacts.